Rules/CommonScripts/plants_regrow.as

```diff
@@ -0,0 +1,199 @@
+#define SERVER_ONLY
+
+const bool moss_stone = false; //should stone turn into mossy stone
+
+const f32 grass_grow_chance = 10;
+const f32 bush_grow_chance = 1;
+const f32 moss_stone_chance = 2;
+
+const u32 moss_time_min = 300*30*30; //10 minutes = 300*30*30

```

Comment:
This is in ticks, not minutes, so the variable should be named `moss_time_ticks` or just `moss_time`.
Also, how is 300 * 30 * 30 = 10 minutes? It should be 30 * 60 * 10 (ticks a second * number of seconds in a minute * number of minutes).